test/e2e/wdio.base.conf.js

```diff
@@ -1,73 +0,0 @@
-const path = require('path')
-const { unlinkSync, mkdirSync } = require('fs')
-const { getRunId, getTestReportDirectory } = require('../envUtils')
-const { APPLICATION_ID } = require('./lib/helpers/constants')
-
-const reporters = [['spec', { onlyFailures: true }]]
-let logsPath
-
-const testReportDirectory = getTestReportDirectory()
-if (testReportDirectory) {
-  reporters.push([
-    'junit',
-    {
-      outputDir: testReportDirectory,
-      outputFileFormat(options) {
-        return `results-${options.cid}.${options.capabilities.browserName}.xml`
-      },
-    },
-  ])
-  logsPath = path.join(testReportDirectory, 'specs.log')
-} else if (!process.env.LOGS_STDOUT) {
-  logsPath = 'specs.log'
-}
-
-module.exports = {
-  runner: 'local',
-  specs: ['./test/e2e/scenario/**/*.scenario.ts'],
-  maxInstances: 5,
-  logLevel: 'warn',
-  waitforTimeout: 10000,
-  connectionRetryTimeout: 90000,
-  connectionRetryCount: 0,
-  framework: 'jasmine',
-  reporters,
-  jasmineOpts: {
-    defaultTimeoutInterval: 60000,
-  },
-  autoCompileOpts: {
-    tsNodeOpts: {
-      transpileOnly: false,
-      files: true,
-      project: 'test/e2e/tsconfig.json',
-    },
-  },
-  onPrepare() {
-    console.log(
-      `[RUM events] https://app.datadoghq.com/rum/explorer?query=${encodeURIComponent(
-        `@application.id:${APPLICATION_ID} @context.run_id:"${getRunId()}"`
-      )}`
-    )
-    console.log(`[Log events] https://app.datadoghq.com/logs?query=${encodeURIComponent(`@run_id:"${getRunId()}"`)}\n`)
-
-    if (testReportDirectory) {
-      try {
-        mkdirSync(testReportDirectory, { recursive: true })
-      } catch (e) {
-        console.log(`Failed to create the test report directory: ${e.message}`)
-      }
-    }
-
-    if (logsPath) {
-      try {
-        unlinkSync(logsPath)
-      } catch (e) {
-        if (e.code !== 'ENOENT') {
-          console.log(`Failed to remove previous logs: ${e.message}`)
-        }
-      }
-    }
-  },
-
-  logsPath,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏FMU, webdriverio don't support the `logsPath` config anymore (or maybe it never did), so you removed from the configuration.

But this was used in `logger.ts` : https://github.com/DataDog/browser-sdk/pull/2135/files#diff-664ed702d3aa8efafdb821a59408d16dd1dffcea3f155a9b2ee750d12c18bdfcL4
Now that it is always undefined, logs will always be sent to stdout, which is a bit verbose.

Maybe we can restore the original behavior by having an "initLogger" function in `lib/framework/logger` that initializes the `stream`, which would be called in `onPrepare` after creating the logs file:

```ts
function initLogger(logsPath?: string) {
  stream = logsPath ? fs.createWriteStream(logsPath, { flags: 'a' }) : process.stdout
}
```